megamek/src/megamek/client/bot/princess/BasicPathRanker.java

```diff
@@ -647,6 +650,15 @@ protected RankedPath rankPath(MovePath path, IGame game, int maxRange,
 
     }
 
+    /**
+     * Worker function that determines if a given enemy entity should be evaluated as if it has moved.
+     */
+    protected boolean evaluateAsMoved(Entity enemy) {
+        // Aerospace units on ground maps can go pretty much anywhere they want, so it's
+        // somewhat pointless to try to predict their movement.
+        return !enemy.isSelectableThisTurn() || enemy.isImmobile() || enemy.isAero();
```

Comment:
Do you want to do a check for airborne here? Since aerodyne on the ground move like tanks, and spheroids are even more predictable. (This is not a request for a change, just a question.)

Reply:
Yes, it looks like I missed a spot here.